carbon-design-system/carbon-addons-iot-react
# fix(datetimepickerv2): add onClear callback

Closes #3760 

**Summary**

- Before we don't have onClear call back for user to reset date and time values. 

**Change List (commits, features, bugs, etc)**

- added onClear() callback prop for single select 

**Acceptance Test (how to verify the PR)**

- Go to [single select story](https://deploy-preview-3761--carbon-addons-iot-react.netlify.app/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-datetimepickerv2--single-select)
- Click on calendar button
- Click on clear button
- In action tab, see onClear was being called with empty objects



**Regression Test (how to make sure this PR doesn't break old functionality)**


https://user-images.githubusercontent.com/24279794/228561704-137e8896-1a2a-48e6-a5ca-ccded6d9a33b.mov





**Things to look for during review**
